### ISSUE TITLE
Enable Projectections Endpoint Should Respond with a 200 for Projections that are Enabled

### ISSUE THREAD

**Issue by sbellware** [EventStore/EventStore#501]
The endpoint that allows enabling a projection will respond with a 500 if making the request for an endpoint that is already enabled.

It would seem that this action should reflect idempotence, and respond with a 200.

Example:

## Now Does

```
name="%24by_category"

curl -i -X POST -d '{"msgTypeId":251,"name":"${name}"}' http://127.0.0.1:2113/projection/${name}/command/enable -u admin:changeit
# => 200 OK

curl -i -X POST -d '{"msgTypeId":251,"name":"${name}"}' http://127.0.0.1:2113/projection/${name}/command/enable -u admin:changeit
# => 500 Failed
```

## Maybe Should Do

```
name="%24by_category"

curl -i -X POST -d '{"msgTypeId":251,"name":"${name}"}' http://127.0.0.1:2113/projection/${name}/command/enable -u admin:changeit
# => 200 OK

curl -i -X POST -d '{"msgTypeId":251,"name":"${name}"}' http://127.0.0.1:2113/projection/${name}/command/enable -u admin:changeit
# => 200 OK
```

**Comment by jen20** [comment]
:+1:

**Comment by pgermishuys** [comment]
Probably want the same for Disable?

**Comment by sbellware** [comment]
I personally haven't tried it with Disable. I'm about to get there. But it would make sense (at least from my perspective as a user) that it would portray idempotence.

**Comment by jen20** [comment]
:+1: same for disable and any other operations which appear idemopotent

**Comment by gregoryyoung** [comment]
This seems quite reasonable.

**Comment by pgermishuys** [comment]
This was merged into 3.0.6
